examples/applications/svm_gui.py

```diff
@@ -144,11 +149,14 @@ def __init__(self, root, controller):
         ax.set_xlim((x_min, x_max))
         ax.set_ylim((y_min, y_max))
         canvas = FigureCanvasTkAgg(f, master=root)
-        canvas.show()
+        try:
+            canvas.show()
```

Comment:
I would prefer to call `draw` first since it is the API in the newer versions of matplotlib.

Add a comment here saying that the `show` call is for supporting matplotlib `1.*`. (Looks like it was change to `draw` in `2.*`.)